src/ProxyManager/Generator/MethodGenerator.php

```diff
@@ -21,10 +23,85 @@ class MethodGenerator extends ZendMethodGenerator
     public static function fromReflection(MethodReflection $reflectionMethod) : self
     {
         /* @var $method self */
-        $method = parent::fromReflection($reflectionMethod);
+        $method = self::fromReflectionWithoutDocBlock($reflectionMethod);
 
         $method->setInterface(false);
 
         return $method;
     }
+
+    public static function fromReflectionWithoutDocBlock(MethodReflection $reflectionMethod)
```

Comment:
Those methods are basically copied from the parent with the phpdoc logic removed. The two private methods needed to be copied as well. It's not that nice. Maybe we can ask zend code to add an option for this or provide another extension point.

Reply:
Isn't it easier to call tge parent method, then set the internal docblock generator?